packages/curl.rb

```diff
@@ -219,8 +219,8 @@ def self.install
 
   def self.postinstall
     if ARCH == 'i686'
-      puts 'Static curl is broken on i686, so use dynamic curl from libcurl package.'.orange
-      FileUtils.cp "#{CREW_PREFIX}/bin/curl.nonstatic", "#{CREW_PREFIX}/bin/curl"
+      puts 'Static curl is broken on i686, so using curl from the libcurl package.'.orange
+      FileUtils.cp_r "#{CREW_PREFIX}/bin/curl.nonstatic", "#{CREW_PREFIX}/bin/curl", remove_destination: true
```

Comment:
What is recursive here?  Maybe `cp_f` is what you need to force overwrite?

Reply:
`cp_f` doesn't exist, as I discovered. See https://ruby-doc.org/stdlib-2.4.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-cp